qa/L0_backend_python/io/io_test.py

```diff
@@ -101,57 +100,56 @@ def _run_ensemble_test(self):
                             self.assertTrue(np.all(output0 == input0))
 
     def test_ensemble_io(self):
-        # Only run the shared memory leak detection with the default trial
-        if TRIAL == "default":
-            with self._shm_leak_detector.Probe():
-                self._run_ensemble_test()
-        else:
-            self._run_ensemble_test()
```

Comment:
This was previously skipping decoupled trial - trying to add decoupled test back.

Reply:
@Tabrizian Do you know why we were skipping shm leak detector probe on decoupled trial? It seems like the decoupled trial will always leak the exact same amount as @rmccorm4 found.

Reply:
Based on our slack conversation I think you found the root cause. And just to set the record straight it is not really a leak since the memory will be deallocated on next inference.